grails-app/services/au/org/ala/volunteer/ProjectService.groovy

```diff
@@ -351,7 +358,7 @@ class ProjectService {
         makeSummaryListFromConditions(conditions, tag, q, sort, offset, max, order, statusFilter, activeFilter, false)
     }
 
-    private def makeSummaryListFromConditions(Collection<? extends Condition> conditions, String tag, String q, String sort, Integer offset, Integer max, String order, ProjectStatusFilterType statusFilter, ProjectActiveFilterType activeFilter, boolean countUser) {
+    private def makeSummaryListFromConditions(Collection<? extends Condition> conditions, def tag, String q, String sort, Integer offset, Integer max, String order, ProjectStatusFilterType statusFilter, ProjectActiveFilterType activeFilter, boolean countUser) {
```

Comment:
Why are we losing the type on `tag` here?

Reply:
tag is now a list of projects. I'll rename the param